### PR TITLE
Adding example code for converting an Article Label into a Content Tag

### PR DESCRIPTION
**Pull request by kaspersorensen**
Using Python and our REST API

### CODE REVIEW

File: article-label-to-content-tag/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Zendesk Gather example API client for user contribution stats
+
+This is a simple utility built in Python that converts an Article Label into a public-facing Content Tags
```
Comment (jrodewig) [inline article-label-to-content-tag/README.md]:
```suggestion
This is a simple utility built in Python that converts an [article label](https://support.zendesk.com/hc/en-us/articles/4408835056154) into a public-facing [content tag](https://support.zendesk.com/hc/en-us/articles/4848925672730).
```

File: article-label-to-content-tag/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+```
+pip install requests
+```
+
+## Usage
+
+This is a command line tool. Invoke it like this:
+
+```
+python main.py -d <subdomain> -u <username> -t <token> -o <output_file> --from=<from> --to=<to>
```
Comment (jrodewig) [inline article-label-to-content-tag/README.md]:
```suggestion
python main.py -d {subdomain} -u {email_address} -t {api_token}  --label {article_label} --tag {content_tag}
```

I couldn't get the `-o` flag to work. I also updated some of these so they're more aligned with the rest of our docs.

File: article-label-to-content-tag/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+Where:
+
+ * `<subdomain>` is your zendesk subdomain
+ * `<username>` is your username
+ * `<token>` is your API token
+ * `<label>` is the name of the label used on articles
+ * `<tag>` is the name of the tag to apply
```
Comment (jrodewig) [inline article-label-to-content-tag/README.md]:
```suggestion
 * `{subdomain}` is your Zendesk subdomain
 * `{email_address}` is your email address for Zendesk login
 * `{api_token}` is your Zendesk API token
 * `{article_label}` is the name of the label used on articles
 * `{content_tag}` is the name of the content tag to apply. This tag must exist before running the tool.
```

File: article-label-to-content-tag/.gitignore [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.vscode/
+__pycache__/
+*.pyc
+build/
+dist/
+.eggs/
+*.egg-info/
+*.dist-info/
```
Comment (jrodewig) [inline article-label-to-content-tag/.gitignore]: I think we can drop this file.

File: article-label-to-content-tag/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Zendesk Gather example API client for user contribution stats
```
Comment (jrodewig) [inline article-label-to-content-tag/README.md]:
```suggestion
# Convert an article label to a content tag
```

**Review by jrodewig — COMMENTED** [review]
Looks good overall. I had a few comments on the README.

File: article-label-to-content-tag/.gitignore (hunk 4 above)
Comment (kaspersorensen) [reply article-label-to-content-tag/.gitignore]: It's not important to me, I just like to have .gitignore files so I don't have to worry about adding IDE metadata or build artifacts etc to the repo.

File: article-label-to-content-tag/README.md (hunk 2 above)
Comment (kaspersorensen) [reply article-label-to-content-tag/README.md]: Doh this was a bad copy-paste from a previous python script I had

File: article-label-to-content-tag/.gitignore (hunk 4 above)
Comment (jrodewig) [reply article-label-to-content-tag/.gitignore]: Sound good. Let's leave it in.

**Comment by kaspersorensen** [comment]
Thanks a lot for the review @jrodewig! I've applied your suggestions and removed the .gitignore file.
I don't think I have merge privileges to this repo so maybe you can either grant me that, or merge the PR for me :)

**Review by jrodewig — APPROVED** [review]

**Comment by jrodewig** [comment]
Sounds good. I'll merge this in. Thanks @kaspersorensen.

**Pull request merged**